diff --git a/lib/make-spawn-args.js b/lib/make-spawn-args.js
index 8a32d7198cb2e2969576ca8fa226926a5fe0afef..1c08440155ec49e2ccc7ee0e6e71bf8ec341aef4 100644
--- a/lib/make-spawn-args.js
+++ b/lib/make-spawn-args.js
@@ -1,7 +1,7 @@
 /* eslint camelcase: "off" */
 const setPATH = require('./set-path.js')
 const { resolve } = require('path')
-const npm_config_node_gyp = require.resolve('node-gyp/bin/node-gyp.js')
+// const npm_config_node_gyp = require.resolve('node-gyp/bin/node-gyp.js')
 
 const makeSpawnArgs = options => {
   const {
@@ -23,7 +23,7 @@ const makeSpawnArgs = options => {
     npm_package_json: resolve(path, 'package.json'),
     npm_lifecycle_event: event,
     npm_lifecycle_script: cmd,
-    npm_config_node_gyp,
+    // npm_config_node_gyp,
   })
 
   const spawnOpts = {
