diff --git a/dist/index.mjs b/dist/index.mjs
index ff38b5e4f5682dab52dd09899f118042ae0a1b36..8192c3d07db81d5557a90b7d5f04498c88f3befa 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1,4 +1,4 @@
-export { d as downloadTemplate, r as registryProvider, s as startShell } from './shared/giget.OCaTp9b-.mjs';
+export { d as downloadTemplate, r as registryProvider, s as startShell, p as providers } from './shared/giget.OCaTp9b-.mjs';
 import 'node:fs/promises';
 import 'node:fs';
 import 'assert';
diff --git a/dist/shared/giget.OCaTp9b-.mjs b/dist/shared/giget.OCaTp9b-.mjs
index 0f909d38f599397cd2a14b6b399399ef93ebd2e6..ef03d803b85f77944b469ff56ed4eb3dafe339c5 100644
--- a/dist/shared/giget.OCaTp9b-.mjs
+++ b/dist/shared/giget.OCaTp9b-.mjs
@@ -465,4 +465,4 @@ async function downloadTemplate(input, options = {}) {
   };
 }
 
-export { downloadTemplate as d, registryProvider as r, startShell as s };
+export { downloadTemplate as d, registryProvider as r, startShell as s, providers as p };
