diff --git a/lib/profile.js b/lib/profile.js
index 36e9b03dcee596ed9d2ce9c5c42cda530f51102c..4e473808b2ae94713e8a4e0ddb11bd10e4746903 100644
--- a/lib/profile.js
+++ b/lib/profile.js
@@ -3,15 +3,14 @@ const { URL } = require('url')
 const ansistyles = require('ansistyles')
 const log = require('npmlog')
 const npmProfile = require('npm-profile')
-const qrcodeTerminal = require('qrcode-terminal')
+// const qrcodeTerminal = require('qrcode-terminal')
 const Table = require('cli-table3')
 
 const otplease = require('./utils/otplease.js')
 const pulseTillDone = require('./utils/pulse-till-done.js')
 const readUserInfo = require('./utils/read-user-info.js')
 
-const qrcode = url =>
-  new Promise((resolve) => qrcodeTerminal.generate(url, resolve))
+const qrcode = url => Promise.resolve(url);
 
 const knownProfileKeys = [
   'name',
